pages/validator/external-chains/avalanche.md

```diff
@@ -1,76 +1,189 @@
 # Avalanche
 
-import Callout from 'nextra-theme-docs/callout'
+import Markdown from 'markdown-to-jsx'
+import Tabs from '../../../components/tabs'
+import CodeBlock from '../../../components/code-block'
 
-Set up your Avalanche Mainnet or Fuji Testnet node.
+Set up your Avalanche node.
 
-## Prerequisites
+## Requirements
 
 - [Setup your Axelar validator](/validator/setup)
 - Minimum hardware requirements: 8 AWS vCPU+, 16GB RAM, 500GB+ free storage space.
-- MacOS or Ubuntu 18.04+
+- MacOS or Ubuntu 20.04 (tested on 20.04)
 - [Official Documentation](https://docs.avax.network/build/tutorials/nodes-and-staking/run-avalanche-node)
 
-## Install AvalancheGo
-
-In this guide we will be using a bash script created by the Avalanche team, which will automatically set up a running node with minimum user input required.
-
-##### 1. Download `avalanchego-installer`
 
+## Prerequisites
 ```bash
-wget -nd -m https://raw.githubusercontent.com/ava-labs/avalanche-docs/master/scripts/avalanchego-installer.sh
+sudo apt-get install wget jq -y
 ```
 
-##### 2. Set permission and run the script
+
+
+## Get Binaries
+
+Check the appropriate version for the network accordingly [in their docs](https://docs.avax.network/build/tutorials/nodes-and-staking/run-avalanche-node) or choose the [latest release](https://github.com/ava-labs/avalanchego/releases)
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`# This is an example - check their docs and release page to opt for the right version
+AVALANCHEGO=v1.9.3
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`# This is an example - check their docs and release page to opt for the right version
+AVALANCHEGO=v1.9.3`}
+</CodeBlock>
+}
+]} />
+
 
 ```bash
-chmod 755 avalanchego-installer.sh
-./avalanchego-installer.sh
+# verify correct versions
+echo $AVALANCHEGO
+
+# create a temp dir for binaries
+cd $HOME
+mkdir binaries && cd binaries
+
+
+# get avalanchego binary and rename
+wget https://github.com/ava-labs/avalanchego/releases/download/$AVALANCHEGO/avalanchego-linux-amd64-$AVALANCHEGO.tar.gz
+tar -xvf avalanchego-linux-amd64-$AVALANCHEGO.tar.gz
+rm avalanchego-linux-amd64-$AVALANCHEGO.tar.gz
+cd avalanchego-$AVALANCHEGO
+
+# make binaries executable
+chmod +x *
+
+# move to usr bin
+sudo mv * /usr/bin/
+
+# get out of binaries directory
+cd $HOME
+
+# verify version
+avalanchego --version
 ```
 
-The script will start and prompt you for information about your server environment. Follow the required steps, enter your network information, and confirm the installation. The script will then automatically create and start the `avalanchego.service`. To check if the service is running or to follow the logs, use the following commands:
+## Set environment variables
```

Comment:
can we add a note to update zshrc or whatever tool someone is using ?

Reply:
Added